code/resources/js/components/Footer.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+    <div>
+    <div class="footer">
+            <div class="content has-text-centered">
+                <p>
+                    <strong>MySuperTasker</strong> by <a href="https://github.com/Niko998" color="black">Nikodem Kabsch</a>.
+                </p>
+            </div>
+        </div>
+        </div>
+</template>
+
+<style>
```

Comment:
poczytaj sobie o "scoped styles" we vue.

Reply:
Chodzi o to, ze w poszczegolnych componentach powinienem uzywac scooped, zeby uniknac ewentualnej kolizji nazw klas miedzy komponentami?